src/Models/ArtistsModel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+require_once 'src/DatabaseConnector.php';
+
+class ArtistsModel
```

Comment:
Missing return types in here - also add the return comments as per my other comment in the album model